docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -80,6 +80,14 @@ The Einstein logo on the activity bar and in the status bar confirms that the ex
 
 ![einstein installed](./images/einstein-installed.png)
 
+## Enable or Disable Einstein For Developers
+
+You can enable or disable Einstein for Developers from within Visual Studio Code. The Einstein for Developers status icon in the bottom panel of the Visual Studio Code window indicates whether Einstein for Developers is enabled or disabled. When enabled, the background color of the icon will match the color of the status bar. When disabled, the background color of the icon is brown. The background color of the icon is red when you're connected to an org that doesn't have Einstein for Developers enabled.
```

Comment:
This part might be a bit easier to understand with images for each of the status options?

Reply:
Agree. I also think we should mention the need to reload the window to refresh the status. Here's the [details](https://salesforce-internal.slack.com/archives/C05822N4J95/p1700264797571319?thread_ts=1700179717.179469&cid=C05822N4J95) I brought up before.

Reply:
I've updated the doc to include text about reloading windows. Can I please get the images from you for each status option so I can add those? Thanks in advance, @AnanyaJha.